mdoc-lsp/src/main/scala/mdoc/internal/lsp/LspLogger.scala

```diff
@@ -8,8 +8,7 @@ import scribe.Logger
 import scribe.format._
 import scribe.writer.Writer
 
-/**
-  * Scribe logging handler that forwards logging messages to the LSP editor client.
+/** Scribe logging handler that forwards logging messages to the LSP editor client.
```

Comment:
Do you all have a preference on this? While valid, I know some may prefer using the `docstrings.blankFirstLine = "yes"` which would preserve the blank line on top. Thoughts?

Reply:
Both are really fine for me, no particular preference.

Reply:
Alright, same for me. I'll go ahead and just merge then.